yolov5_rknn_coco:
  image_topic_name: /ascamera_hp60c/rgb0/image
  
  model_url: /home/orangepi/Documents/yolov5_weights/coco_s_rk3588s.rknn
  labels_url: /home/orangepi/Documents/yolov5_weights/labels_coco.txt
  
  class_num: 80
  anchors: [10, 13, 16, 30, 33, 23, 30, 61, 62, 45, 59, 119, 116, 90, 156, 198, 373, 326]
  
  conf_thresh: 0.5
  iou_thresh: 0.5

  kill_switch_topic: /task

  byte_track:
    target_class_id: 0
    track_thresh: 0.4
    high_thresh: 0.5
    match_thresh: 0.7
    track_buffer: 60
